package.json

```diff
@@ -115,6 +115,7 @@
     "search-index": "3.4.0",
     "sinon": "13.0.1",
     "typescript": "4.8.2",
+    "typescript-strict-plugin": "^2.1.0",
```

Comment:
Are you adding this dependency temporarily until we have moved everything to stricter checks?

An alternative approach to make progress for #261 would be to turn `"strict": true` on in `tsconfig` and fix what you can/want then comment out the rule again.

I just do now want:
1. for this dependency to be permanent; and
1. for us to add every file in the `paths` in both `tsconfig` files in order to complete #261, turning on the strict rule itself seems to be easier to be sure and know the actual progress.

Reply:
Yes, the intention was to use this temporarily until we finish migrating to strict. I don't feel very strongly about this approach, so turning on `"strict": true` while developing works for me.